src/components/LongitudinalSection.js

```diff
@@ -77,6 +97,34 @@ function Longitudinal({ selectedSection, coverageData }) {
     },
   ];
 
+  const fields = getAllFieldCoveredCounts(coverageData);
+  const sectionFractions = {
+    [patientSectionId]: `${
+      fields.filter((field) => field.section === patientSectionId && field.percentage === 1).length
+    }/${fields.filter((field) => field.section === patientSectionId).length}`,
+    [outcomeSectionId]: `${
+      fields.filter((field) => field.section === outcomeSectionId && field.percentage === 1).length
+    }/${fields.filter((field) => field.section === outcomeSectionId).length}`,
+    [diseaseSectionId]: `${
+      fields.filter((field) => field.section === diseaseSectionId && field.percentage === 1).length
+    }/${fields.filter((field) => field.section === diseaseSectionId).length}`,
+    [treatmentSectionId]: `${
+      fields.filter((field) => field.section === treatmentSectionId && field.percentage === 1).length
+    }/${fields.filter((field) => field.section === treatmentSectionId).length}`,
+    [assessmentSectionId]: `${
+      fields.filter((field) => field.section === assessmentSectionId && field.percentage === 1).length
+    }/${fields.filter((field) => field.section === assessmentSectionId).length}`,
+    [genomicsSectionId]: `${
+      fields.filter((field) => field.section === genomicsSectionId && field.percentage === 1).length
+    }/${fields.filter((field) => field.section === genomicsSectionId).length}`,
+    [overallSectionId]: `${
+      fields.filter((field) => field.section === overallSectionId && field.percentage === 1).length
+    }/${fields.filter((field) => field.section === overallSectionId).length}`,
+  };
+
+  const sectionPercentage = sectionPercentages[selectedSection](coverageData);
+  const percentage = Math.round(sectionPercentage.percentage * 100).toLocaleString();
```

Comment:
We round to 2 decimal places in the main visualization, I think it would be good to keep that consistent across the app.